packages/react-components/react-nav-preview/src/components/NavDrawer/renderNavDrawer.tsx

```diff
@@ -1,22 +1,11 @@
 /** @jsxRuntime automatic */
 /** @jsxImportSource @fluentui/react-jsx-runtime */
```

Comment:
These can be removed
```suggestion
```

Reply:
I get this error when I do that:

![image](https://github.com/microsoft/fluentui/assets/17346018/2021f2d8-b29a-47c8-9b51-41391786248d)

Reply:
recapping offline - skipping this for now.